extensions/amp-story/1.0/amp-story-open-page-attachment.js

```diff
@@ -108,6 +133,67 @@ const renderDefaultPageAttachmentUI = (pageEl, attachmentEl) => {
   return openAttachmentEl;
 };
 
+/**
+ * Renders inline page attachment UI.
+ * @param {!Window} win
+ * @param {!Element} pageEl
+ * @param {!Element} attachmentEl
+ * @param {!Element} attachmentHref
+ * @return {!Element}
+ */
+const renderOutlinkPageAttachmentUI = (
+  win,
+  pageEl,
+  attachmentEl,
+  attachmentHref
+) => {
+  const openAttachmentEl = buildOpenOutlinkAttachmentElement(pageEl);
+
+  // Copy href to the element so it can be previewed on hover and long press.
+  if (attachmentHref) {
+    openAttachmentEl.setAttribute('href', attachmentHref);
+  }
+
+  // Setting theme
+  openAttachmentEl.setAttribute('theme', attachmentEl.getAttribute('theme'));
+
+  // Getting elements
+  const {chipEl, ctaLabelEl} = htmlRefs(openAttachmentEl);
+
+  // Appending text & aria-label.
+  const openLabelAttr = attachmentEl.getAttribute('data-cta-text');
+  const openLabel = openLabelAttr
+    ? openLabelAttr.trim()
+    : getLocalizationService(pageEl).getLocalizedString(
+        LocalizedStringId.AMP_STORY_PAGE_ATTACHMENT_OPEN_LABEL
+      );
+  ctaLabelEl.textContent = openLabel;
+  openAttachmentEl.setAttribute('aria-label', openLabel);
+
+  // Adding image.
+  const openImgAttr = attachmentEl.getAttribute('cta-image');
+
+  if (openImgAttr === 'none') {
```

Comment:
Could be useful to add a comment here explaining "removes image if attribute is explicitly set to none".

Also, you can add this class on the parent element `.i-amphtml-story-page-open-attachment` instead of adding it in both places right?

Reply:
The text and the chip need diff padding styling. But I figured out how to do it with just one additional class by using:

```
.i-amphtml-story-outlink-page-attachment-outlink-chip-no-image {
  padding-left: 14px !important;
}

.i-amphtml-story-outlink-page-attachment-outlink-chip-no-image > * {
  padding-left: 0px !important;
}
```